corehq/apps/user_importer/helpers.py

```diff
@@ -174,20 +174,16 @@ def update_user_data(self, data, uncategorized_data, profile_name, profiles_by_n
         if user_data.profile_id and user_data.profile_id != old_profile_id:
             self.logger.add_info(UserChangeMessage.profile_info(user_data.profile_id, profile_name))
 
+        if old_user_data != user_data.raw:
+            self.logger.add_changes({'user_data': user_data.raw}, skip_confirmation=True)
```

Comment:
Can you elaborate?  This code logs all changes made to a user for auditing purposes.

Reply:
Maybe it is just the name of the field being `user_data` and in actuality it does not contain anything sensitive. But I wonder if there was a chance something that we don't want logged ends up in there. I think it would not be clear to every developer changing what goes in `user_data` that it will also end up in the logs.

Reply:
true - though `logger` here actually isn't a normal python logger, it's an instance of `UserChangeLogger`, which saves details of changes to postgres to power an auditing report.

Reply:
Fair